ui/lib/kmip/addon/templates/credentials/show.hbs

```diff
@@ -49,13 +49,7 @@
         <A.Title>Warning</A.Title>
         <A.Description>You will not be able to access the private key later, so please copy the information below.</A.Description>
       </Hds::Alert>
-      <MaskedInput
-        @value={{this.model.privateKey}}
-        @name="Private key"
-        @allowCopy={{true}}
-        @displayOnly={{true}}
-        class="is-block"
-      />
+      <MaskedInput @value={{this.model.privateKey}} @name="Private key" @allowCopy={{true}} @displayOnly={{true}} />
```

Comment:
Remove `is-block` class 
### before:
<img width="1116" alt="Screenshot 2023-09-12 at 9 36 20 AM" src="https://github.com/hashicorp/vault/assets/68122737/85531d2c-b46b-436e-a737-e88ae93fa90b">

<hr>

### after
<img width="1057" alt="Screenshot 2023-09-12 at 9 35 26 AM" src="https://github.com/hashicorp/vault/assets/68122737/a882ffb4-1db5-45bc-8bd8-6941af1cea4b">